src/vs/platform/protocol/electron-main/protocolMainService.ts

```diff
@@ -104,6 +104,12 @@ export class ProtocolMainService extends Disposable implements IProtocolMainServ
 			}
 		}
 
+		// Disable in-memory cache when running out of sources
+		if (process.env['VSCODE_DEV']) {
+			headers = headers || {};
+			headers['Cache-Control'] = 'no-cache';
```

Comment:
`COI.CoopAndCoep` uses `Object.freeze`, is this going to throw a runtime exception?